### PR TITLE
feature(plugin): Config Manager plugin

### PR DESCRIPTION
**Pull request by leosmerling-hopeit**
Config Manager:
============
Plugin to query runtime configuration of single and multiple servers, in order to enable monitoring of multiple hosts.

To be used by apps-visualizer plugin in future releases.

Check [README](../blob/b477f66e212139ffe0eba1bf5fb04dca07253f98/plugins/ops/config-manager/README.md) for usage details.

### Try it out using simple-example app

```
make install-simple-example
```

-Run multiple `hopeit_server` instances using different ports and including simple-example app and this plugin:

```
nohup make PORT=8020 run-simple-example &
nohup make PORT=8021 run-simple-example &
```

To get running config for a single server:

```
curl localhost:8020/api/config-manager/0x7/runtime-apps-config
```

To get combined running apps config from a group of servers (cluster):

```
curl localhost:8020/api/config-manager/0x7/cluster-apps-config?hosts=http://localhost:8020,http://localhost:8021
```

### CODE REVIEW

**Review by leosmerling-hopeit — COMMENTED** [review]
Self-review done.

**Review by pcanto-hopeit — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -103,3 +103,21 @@ pypi-test:
 pypi-test-plugin:
 	pip install twine && \
 	python -m twine upload -u=__token__ -p=$(TEST_PYPI_API_TOKEN) --repository testpypi $(PLUGINFOLDER)/dist/*
+
+install-simple-example:
+	make install && \
+	make PLUGINFOLDER=plugins/streams/redis install-plugin && \
+	make PLUGINFOLDER=plugins/storage/fs install-plugin && \
+	make PLUGINFOLDER=plugins/ops/config-manager install-plugin && \
```
Comment (pcanto-hopeit) [inline Makefile:111]: missing:
	make PLUGINFOLDER=plugins/storage/redis install-plugin && \

**Review by pcanto-hopeit — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (leosmerling-hopeit) [reply Makefile:111]: Is redis-storage needed in simple-example?

Comment (pcanto-hopeit) [reply Makefile:111]: no, it is not! :)

**Review by pcanto-hopeit — APPROVED** [review]

**Pull request merged**